resources/uk/gov/hmcts/helm/push-helm-charts-to-git.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -x
+
+CHART_DIRECTORY=${1}
+CHART_NAME=${2}
+VERSION=${3}
+
+RETRIES=3
+DELAY=5
+COUNT=0
+
+if [ -d "hmcts-charts" ]; then
+  rm -rf hmcts-charts
+fi
+
+while [ $COUNT -lt $RETRIES ]; do
+  if git clone https://github.com/hmcts/hmcts-charts.git; then
+    COUNT=0
+    break
+  fi
+  COUNT=$(( $COUNT + 1))
+  if [ "$COUNT" = $RETRIES ]; then
+    exit 1
+  fi
+  sleep $DELAY
+done
+
+if cd hmcts-charts; then
+  if [ -d "${CHART_NAME}" ]; then
+    GIT_CHART_VERSION=$(helm inspect chart "${CHART_NAME}" | grep ^version | cut -d  ':' -f 2 | tr -d '[:space:]')
+    if [ "${GIT_CHART_VERSION}" = "${VERSION}" ]; then
+      echo "No differences in the charts. Nothing to publish" 1>&2
+      exit 0
+    fi
+  else
+     echo "Chart version ${GIT_CHART_VERSION} not published to git yet" 1>&2
+  fi
+  
+  if cp -R "../${CHART_DIRECTORY}" ./stable/; then
+    git remote set-url origin $(git config remote.origin.url | sed "s/github.com/${GIT_CREDENTIALS_ID}:${BEARER_TOKEN}@github.com/g")
+    git config --global user.name "${GIT_CREDENTIALS_ID}"
+    git config --global user.email "${GIT_APP_EMAIL_ID}"
+    git add "${CHART_NAME}/"
+    git commit -m "chart push to git"
```

Comment:
can we have a better commit message please?

taking flux as an example:
https://github.com/hmcts/cnp-flux-config/commit/791e78b092b6d850de7ac3bacc28aae2725b5984

possibly?
`Auto-release cnp-plum-frontend <version>`